devel/code_generation.py

```diff
@@ -126,41 +146,56 @@ def split_into_sections(code: str) -> List[SectionInfo]:
 def combine_sections(sections: List[SectionInfo]) -> str:
     return '\n'.join([s.code for s in sections])
 
-class ExtensionInfo(NamedTuple):
-    name: str
-    relPath: str
-
-def get_extensions(realpath: str, relpath: str) -> List[ExtensionInfo]:
+def get_extensions(realpath: str, relpath: str) -> List[Dict]:
```

Comment:
modified this to get the tags -- this info is then passed to the jinja2 templater